tensorboard/components/tf_tensorboard/test/tensorboardTests.ts

```diff
@@ -23,11 +27,14 @@ namespace tf_tensorboard {
       });
 
       it('renders injected content', function() {
-        let injected = tensorboard.querySelector('#inject-me');
-        chai.assert.isNotNull(injected);
+        let overview = tensorboard.querySelector('#custom-overview');
+        expect(overview.assignedSlot.name).to.equal('injected-overview');
+
+        let headerItems = tensorboard.querySelector('#custom-header-items');
+        expect(headerItems.assignedSlot.name).to.equal('injected-header-items');
       });
 
-      it('reloads the active dashboard on request', (done) => {
+      xit('reloads the active dashboard on request', (done) => {
```

Comment:
prefer to accompany skipped tests with a reason.

Reply:
I've added a comment

Reply:
xdescribe, too. if you think they are stale, maybe just omit.

Reply:
Done - added to xdescribe